### PR TITLE
97: switch to collection chain

### LINKED ISSUE #97: Notify user when they're on the wrong chain

### PR DESCRIPTION
**Pull request by piersss**
# Description

Fixes #97 

https://github.com/airswap/airswap-marketplace/assets/86911296/14451772-8770-46a1-99c7-dce1076e76a9

# Steps to test

Example:

1. Switch back and forth from mainnet to goerli 

# Checklist:

- [x] I have read and understood [CONTRIBUTING.MD](https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md)
- [x] I have included the ticket number in my PR
- [x] I have performed a self-review of my own code
- [x] I have commented the parts where my code is hard to understand

### CODE REVIEW

File: src/redux/stores/balances/balancesSubscriber.ts [hunk 1]
```diff
@@ -15,6 +20,17 @@ export const configureBalancesSubscriber = () => {
     const lastTransaction = transactions.transactions[0];
     const lastSucceededTransaction = lastTransaction?.status === 'succeeded' ? lastTransaction : undefined;
 
+    if (web3.chainId && chainId !== web3.chainId && web3.chainId !== config.chainId) {
```
Comment (filipmanole) [inline src/redux/stores/balances/balancesSubscriber.ts:23]: Why is `chainId` stored in balance store? If it is a duplicate of `chainId` from web3 store should we remove one of them?

File: src/redux/stores/config/configSlice.ts [hunk 2]
```diff
@@ -18,6 +17,8 @@ const initialState: ConfigState = {
   ipfcGatewayURL: process.env.REACT_APP_IPFC_GATEWAY_URL || 'https://ipfs.io/ipfs/',
 };
 
+console.log(initialState);
```
Comment (filipmanole) [inline src/redux/stores/config/configSlice.ts:20]: Is this console.log left on purpose or should be deleted?

File: src/widgets/ListNftWidget/ListNftWidget.tsx [hunk 3]
```diff
@@ -24,6 +26,7 @@ const ListNftWidget: FC<ListNftWidgetProps> = ({ className = '' }) => {
     && currencyTokenInfo
```
Comment (filipmanole) [inline src/widgets/ListNftWidget/ListNftWidget.tsx:26]: `chainId` verification from this test (line15) is already included in verification for `chainIdIsCorrect`

**Review by filipmanole — APPROVED** [review]
Tested locally, it worked flawlessly! 🌮

File: src/redux/stores/balances/balancesSubscriber.ts (hunk 1 above)
Comment (piersss) [reply src/redux/stores/balances/balancesSubscriber.ts:23]: It's not stored in balances store. Subscriber only listens to changes in store, and the chainId is only stored in the function.

File: src/redux/stores/config/configSlice.ts (hunk 2 above)
Comment (piersss) [reply src/redux/stores/config/configSlice.ts:20]: console.log should never be left!

**Pull request merged**